INSTALL.md

```diff
@@ -0,0 +1,328 @@
+  cd build
+  cmake -DENABLE_SPICE=ON ..
+  make -jN`
+
+Instead of N, pass a number of CPU cores you want to use during a build.
+
+Install:
+
+  `make install`
+
+Celestia will be installed into /usr/local by default, with data files landing
+in /usr/local/share/celestia, but you may specify a new location with the
+following option to cmake: `-DCMAKE_INSTALL_PREFIX=/another/path`.
+
+To build the application bundle, pass -DNATIVE_OSX_APP=ON to the cmake command,
+the application bundle will be located in the "build" folder that you previously
+created.
+
+## Supported CMake parameters
+
+List of supported parameters (passed as `-DPARAMETER=VALUE`):
+
+ Parameter            | TYPE | Default | Description
+----------------------| ------|---------|--------------------------------------
+| CMAKE_INSTALL_PREFIX | path | *       | Prefix where to install Celestia
+| CMAKE_PREFIX_PATH    | path |         | Additional path to look for libraries
+| LEGACY_OPENGL_LIBS   | bool | **OFF   | Use OpenGL libraries not GLvnd
+| ENABLE_CELX          | bool | ON      | Enable Lua scripting support
+| ENABLE_SPICE         | bool | OFF     | Enable NAIF kernels support
+| ENABLE_NLS           | bool | ON      | Enable interface translation
+| ENABLE_GLUT          | bool | OFF     | Build simple Glut frontend
+| ENABLE_GTK           | bool | **OFF   | Build legacy GTK2 frontend
+| ENABLE_QT            | bool | ON      | Build Qt frontend
+| ENABLE_WIN           | bool | ***ON   | Build Windows native frontend
+| ENABLE_THEORA        | bool | **ON    | Support video capture to OGG Theora
+| ENABLE_TOOLS         | bool | OFF     | Build tools for Celestia data files
+| ENABLE_TTF           | bool | ****OFF | Build with FreeType support
+| NATIVE_OSX_APP       | bool | OFF     | Support native OSX data paths
+
+Notes:
+ \* /usr/local on Unix-like systems, c:\Program Files or c:\Program Files (x86)
+   on Windows depending on OS type (32 or 64 bit) and build configuration.
+   This option effect is overriden by NATIVE_OSX_APP.
+ ** Ignored on Windows systems.
+ *** Ignored on Unix-like systems.
+ **** This option support is not finished yet.
+
+Parameters of type "bool" accept ON or OFF value. Parameters of type "path"
+accept any directory.
+
+On Windows systems two additonal options are supported:
+- `CMAKE_GENERATOR_PLATFORM` - can be set to x64 on 64-bit Windows to build
+  64-bit Celestia. To build 32-bit Celestia it should be omitted.
+- `CMAKE_TOOLCHAIN_FILE` - location of vcpkg.cmake if vcpkg is used.
+
+
+Executable files
+----------------
+
+As said prevously Celestia provides several user interfaces, accordingly with
+interfaces it's built with it has different executable files installed to
+${CMAKE_INSTALL_PREFIX}/bin (e.g. with default CMAKE_INSTALL_PREFIX on
+Unix-like systems they are installed into `/usr/local/bin`).
+
+Here's the table which provides executable file names accordingly to interface:
+
+ Interface | Executable name
+-  | :-
```

Comment:
either an error or non-standard md extension

Reply:
Non-standard extension. But GitHub supports tables like that.
Else we need another way to make tables. Markdown supports HTML tables, but that's usually not elegant IMHO.

Reply:
No sorry, it is an error.